crossplane-provider-gcp/uptest_changes.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/Makefile b/Makefile
+index 9e0eb0c5..83d001b9 100644
+--- a/Makefile
++++ b/Makefile
+@@ -69,11 +69,8 @@ export SUBPACKAGES := $(SUBPACKAGES)
+ 
+ KIND_VERSION = v0.15.0
+ # dependency for up
+-UP_VERSION = v0.28.0
++UP_VERSION = v0.30.0
+ UP_CHANNEL = stable
+-UPTEST_VERSION = v0.11.1
```

Comment:
Question: Why don't we set he upset_version here to that one too ?

Reply:
The solution came from a PR from the main [repository](https://github.com/crossplane-contrib/provider-upjet-aws/blob/main/Makefile)

Reply:
Good. Thanks.

But I am now wondering why we change the code with the UPTEST_VERSION after looking at the link you shared.